bin/convection

```diff
@@ -15,9 +15,11 @@ module Convection
     end
 
     desc 'converge STACK', 'Converge your cloud'
+    option :stack_group, :type => :string, :desc => 'The name of a stack group defined in your cloudfile to converge'
+    option :stack_list, :type => :string, :desc => 'A ordered comma separated list of stacks to converge'
```

Comment:
@jmanero-r7 @bcall-r7 @onefrankguy @jmanero-r7 @athompson-r7 what do you guys think about this command line option name? I wonder if `--stacks` makes more sense than `--stack-list`.

Respond with a 👍 to this comment for `--stacks` or a 👎 for `--stack-list`.